packages/x-components/src/x-modules/tagging/store/__tests__/actions.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { createLocalVue } from '@vue/test-utils';
+import Vuex, { Store } from 'vuex';
+import { getTaggingResponseStub } from '../../../../__stubs__/tagging-response-stubs.factory';
+import { SearchAdapterDummy } from '../../../../__tests__/adapter.dummy';
+import { installNewXPlugin } from '../../../../__tests__/utils';
+import { SafeStore } from '../../../../store/__tests__/utils';
+import { taggingXStoreModule } from '../module';
+import { TaggingActions, TaggingGetters, TaggingMutations, TaggingState } from '../types';
+import { resetTaggingStateWith } from './utils';
+
+describe('testing tagging module actions', () => {
+  const queryTagging = getTaggingResponseStub();
+  const adapter = SearchAdapterDummy;
+  const localVue = createLocalVue();
+  localVue.config.productionTip = false; // Silent production console messages.
+  localVue.use(Vuex);
+
+  const store: SafeStore<TaggingState, TaggingGetters, TaggingMutations, TaggingActions> =
+    new Store(taggingXStoreModule as any);
+  installNewXPlugin({ adapter, store }, localVue);
+
+  beforeEach(() => {
+    resetTaggingStateWith(store);
+    jest.clearAllMocks();
+  });
+
+  describe('trackTagging', () => {
+    it('should track without session id if the consent is not provided', async () => {
+      await store.dispatch('track', queryTagging);
+
+      //eslint-disable-next-line @typescript-eslint/unbound-method
```

Comment:
Instead of having several `//eslint-disable-next-line @typescript-eslint/unbound-method`, can you put this before and after the global `describe`?

`/* eslint-disable @typescript-eslint/unbound-method */`
`/* eslint-enable @typescript-eslint/unbound-method */`